src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml

```diff
@@ -65,13 +65,13 @@
         }
 
 
-        @if (Model.Data.HasApprovedChangeOfProviderRequest || Model.Data.IsContinuation)
+        @if (Model.Data.HasApprovedChangeOfProviderRequest || Model.Data.IsEmployerContinuation)
         {
 
             <div class="panel panel-border-wide alert-default">
 
 
-                @if (Model.Data.HasApprovedChangeOfProviderRequest && Model.Data.IsContinuation)
+                @if (Model.Data.HasApprovedChangeOfProviderRequest && Model.Data.IsEmployerContinuation)
```

Comment:
line 88, should that be IsEmployerContinuation

Reply:
something, i just noticed all the if condition have the same text - we can remove the nested if else, as one of them is going to be true.